integrations/qdrant/src/haystack_integrations/document_stores/qdrant/document_store.py

```diff
@@ -496,7 +496,7 @@ def _query_by_sparse(
         query_sparse_embedding: SparseEmbedding,
         filters: Optional[Union[Dict[str, Any], rest.Filter]] = None,
         top_k: int = 10,
-        scale_score: bool = True,
+        scale_score: bool = False,
```

Comment:
the changes in this file are not needed, since the only interface to these methods is the Retriever,
but I think they make the overall behavior clearer.